eth/downloader/queue.go

```diff
@@ -39,8 +39,8 @@ import (
 )
 
 var (
-	blockCacheLimit   = 8192 // Maximum number of blocks to cache before throttling the download
-	maxInFlightStates = 4096 // Maximum number of state downloads to allow concurrently
+	blockCacheLimit   = 16384 // Maximum number of blocks to cache before throttling the download
```

Comment:
This is to support the bump of `maxResultsProcess` in the downloader to `4096`. Previously we were importing blocks in batches of 256 which for some reason takes way more time than doing larger batches. As the 256 was an arbitrary amount to support interruption, and since then we've implemented proper interruption, there's no reason to keep 256 if higher values produce much better results.

Among other, larger batches allow smoother POW verification distribution on multiple processors and also give a bit of time for the downloader to fetch the next batch, whereas 256 often quickly returned, the downloader having only a little data due to a laggy peer, which got imported even faster, getting to a scenario where blocks were imported 1 by 1 fast, and not taking advantage of multiple cores.

At least this is the theory. There has not been scientific measurements made to back this claim up, only "seen" perormance changes.